core/src/main/scala/org/broadinstitute/dsde/rawls/billing/BillingRepository.scala

```diff
@@ -0,0 +1,24 @@
+package org.broadinstitute.dsde.rawls.billing
+
+import org.broadinstitute.dsde.rawls.dataaccess.SlickDataSource
+import org.broadinstitute.dsde.rawls.model.{RawlsBillingProject, RawlsBillingProjectName}
+
+import scala.concurrent.Future
+
+/**
+ * Data access for rawls billing projects
+ */
+class BillingRepository(dataSource: SlickDataSource) {
```

Comment:
I added this class to improve testability of the `BillingProjectOrchestrator` class referenced above. That is, I didn't want that class to have to worry about Slick types in it's business logic, which would also force leakage of those types to the related tests.